kat/kat/harness.py

```diff
@@ -1097,6 +1097,9 @@ def _setup_k8s(self, selected):
 
         self._wait(selected)
 
+        print("Waiting 5s after requirements, just because...")
```

Comment:
"just because" :rofl:

Reply:
We'll see if it seems to make anything better over time. 😂